packages/crew_profile_base.rb

```diff
@@ -28,6 +28,12 @@ def self.install
   end
 
   def self.postinstall
+    # Remove install.sh provided path file since we supersede it.
```

Comment:
Put in lib/fixup.rb? Every installation of crew should have crew_profile_base anyways

Reply:
moved!